app/controllers/PspSchemeTaskListController.scala

```diff
@@ -35,13 +36,15 @@ class PspSchemeTaskListController @Inject()(
                                              getData: PspDataRetrievalAction,
                                              val controllerComponents: MessagesControllerComponents,
                                              val view: pspTaskList,
-                                             hsTaskListHelperPsp: HsTaskListHelperPsp
+                                             hsTaskListHelperPsp: HsTaskListHelperPsp,
+                                             @TaskList allowAccessAction: AllowAccessActionProvider
```

Comment:
This allowAccessAction is bound to the TaskList implementation, assuming that it wants to use its params of allowPsa = true, allowPsp = false, but then both values are overridden to true on line 47. Is this binding necessary or should the values change?

Reply:
Reverted allow access changes implemented new SchemeAuthActions that work the same as MPS service.